config/jobs/kubernetes/sig-release/release-team-jobs/release-team-periodics.yaml

```diff
@@ -0,0 +1,30 @@
+periodics:
+- cron: "0 */3 * * *" # Every 3 hours
```

Comment:
Can we add something in here to indicate how one might pause this job after enhancement freeze?

Reply:
Not super familiar with Prow, but is it possible to add in the typical k8s cronjob parameters around concurrency, failurePolicy, deadlineSeconds, etc?

Reply:
> Can we add something in here to indicate how one might pause this job after enhancement freeze?

So, this bit would be more of a process/documentation oriented - i.e. the enhancements lead raise a PR to disable this job after enhancements freeze is enforced. Honestly, it's still an open item to discuss.

> Not super familiar with Prow, but is it possible to add in the typical k8s cronjob parameters around concurrency, failurePolicy, deadlineSeconds, etc?

FMU, no we can't add those parameters to a prow job config.